src/tool/pmix_tool.c

```diff
@@ -870,7 +870,7 @@ PMIX_EXPORT int PMIx_tool_init(pmix_proc_t *proc,
     return rc;
 }
 
-pmix_status_t pmix_tool_init_info(void)
+PMIX_EXPORT pmix_status_t pmix_tool_init_info(void)
```

Comment:
I thought we only put the `PMIX_EXPORT` on the function declaration, not the definition?

Reply:
I guess it follows the pattern from `PMIx_tool_init` above. I'm fine with it, just didn't know.

Reply:
We haven't been consistent, I fear - but I have no strong feelings either way